index.html

```diff
@@ -30,16 +49,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
```

Comment:
To take advantage of cascading and parent/children you could keep the `<div class="box">Box1</div>` and just add an additional class to it `<div class="box box1">Box 1</div>` This would allow you to keep all styles in a parent instead of putting them all in each individual box.